

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <title>Variants annotation fields &#8212; doc 0.0.1 documentation</title>
    <link rel="stylesheet" href="_static/alabaster.css" type="text/css" />
    <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
    <script type="text/javascript">
      var DOCUMENTATION_OPTIONS = {
        URL_ROOT:    './',
        VERSION:     '0.0.1',
        COLLAPSE_INDEX: false,
        FILE_SUFFIX: '.html',
        HAS_SOURCE:  true,
        SOURCELINK_SUFFIX: '.txt'
      };
    </script>
    <script type="text/javascript" src="_static/jquery.js"></script>
    <script type="text/javascript" src="_static/underscore.js"></script>
    <script type="text/javascript" src="_static/doctools.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="prev" title="Getting started" href="getting_started.html" />
   
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
  
  
  <meta name="viewport" content="width=device-width, initial-scale=0.9, maximum-scale=0.9" />

  </head>
  <body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          <div class="body" role="main">
            
  <div class="section" id="variants-annotation-fields">
<h1>Variants annotation fields<a class="headerlink" href="#variants-annotation-fields" title="Permalink to this headline">¶</a></h1>
<p><strong>Location</strong>
The position of variant as <em>chrom:start</em>. The <em>chrom</em> is the chromosome on which the variant resides (from VCF CHROM field) and <em>start</em> is the physical position on the chromosome (1-based coordinate).</p>
<p><strong>Gene</strong>
The gene affected by the variant.</p>
<div class="section" id="genomic-and-genetic-data">
<h2>Genomic and genetic data<a class="headerlink" href="#genomic-and-genetic-data" title="Permalink to this headline">¶</a></h2>
<p><strong>Ref</strong>
Reference allele (from VCF REF field).</p>
<p><strong>Alt</strong>
Alternate allele for the variant (from VCF ALT field).</p>
<p><strong>AA</strong>
The amino acid change, when applicable in <em>ref-pos-alt</em> format. The ‘%3D’ symbol results a deletion.</p>
<p><strong>Zygosity</strong>
The zygosity of the variant in patient. Inferred from the VCF GT genotype tag as:  <em>het</em> for a call of a heterozygous genotype, <em>hom_ref</em> for a call of a homozygous reference genotype and <em>hom_alt</em> for a call of a homozygous alternate (variant) genotype.</p>
<p><strong>HGVS</strong>
Human Genome Variation Sequence c-syntax nomenclature (from VEP).</p>
<p><strong>Genotype</strong>
The sample genotypes.</p>
<p><strong>Genotype Depths</strong>
The depth of aligned sequence observed for each sample.</p>
<p><strong>Genotype Quality</strong>
the genotype quality (PHRED scale) estimates for each sample.</p>
<p><strong>RefSeq</strong>
The transcripts (IDs) relevant to the effect on the protein.</p>
<p><strong>Codon</strong>
What is the codon change?</p>
<p><strong>dbSNP</strong>
A list of rs ids for variants present in dbSNP.</p>
</div>
<div class="section" id="variant-calling-q-r">
<h2>Variant Calling Q&amp;R<a class="headerlink" href="#variant-calling-q-r" title="Permalink to this headline">¶</a></h2>
<p><strong>qual</strong>
Variant confidence or quality by depth.</p>
<p><strong>depth</strong>
The number of aligned sequence reads that led to this variant call.</p>
</div>
<div class="section" id="evidence">
<h2>Evidence<a class="headerlink" href="#evidence" title="Permalink to this headline">¶</a></h2>
<p><strong>Clinvar Phenotypes</strong>
‘,’ delimited list of phenotypes associated with this gene (includes any variant in the same gene in clinvar not just the current variant).</p>
<p><strong>Clinvar significance</strong>
The clinical significance scores for each of the variant according to ClinVar: unknown, untested, non-pathogenic, probable-non-pathogenic,probable-pathogenic, pathogenic, drug-response, histocompatibility, other.</p>
<p><strong>COSMIC</strong>
A list of known COSMIC ids for this variant.</p>
<p><strong>OMIM</strong></p>
<p><strong>MedGen</strong></p>
<p><strong>Orphanet</strong></p>
<p><strong>SNOMED</strong></p>
</div>
<div class="section" id="effect-prediction">
<h2>Effect&amp;Prediction<a class="headerlink" href="#effect-prediction" title="Permalink to this headline">¶</a></h2>
<p><strong>Effect</strong>
The consequence of the most severely affected transcript.</p>
<p><strong>Severity</strong>
Severity of the highest order observed for the variant.</p>
<p><strong>GERP RS</strong>
GERP conservation score (rejected substitution). Higher scores reflect greater conservation.</p>
<p><strong>LRT pred</strong>
Prediction of our LR based ensemble prediction score, “T(olerated)” or “D(amaging)”. The score cutoff between “D” and “T” is 0.5. The rankscore cutoff between “D” and “T” is 0.82268.</p>
<p><strong>MutTaster</strong>
MutationTaster prediction.</p>
<p><strong>Polyphen2 HDIV</strong>
Polyphen2 prediction based on HumDiv.</p>
<p><strong>Polyphen2 HVAR</strong>
Polyphen2 prediction based on HumVar.</p>
<p><strong>SIFT</strong>
SIFT predictions for the snp’s for the most severely affected transcript.</p>
</div>
<div class="section" id="frequency">
<h2>Frequency<a class="headerlink" href="#frequency" title="Permalink to this headline">¶</a></h2>
<p><strong>Max AF</strong>
The maximum of  aaf_gnomad{afr,amr,eas,nfe,sas}, aaf_esp_ea, aaf_esp_aa, aaf_1kg_amr, aaf_1kg_eas, aaf_1kg_sas, aaf_1kg_afr, aaf_1kg_eur, aaf_adj_exac_afr, aaf_adj_exac_amr, aaf_adj_exac_eas, aaf_adj_exac_nfe, aaf_adj_exac_sas, and -1 if none of those databases/populations contain the variant.</p>
<p><strong>1K Genome AF</strong>
Global allele frequency (based on AC/AN) (1000g project - phase 3)</p>
<p><strong>ESP African/American AF</strong>
Minor Allele Frequency of the variant for African Americans in the ESP project</p>
<p><strong>ESP European/American AF</strong>
Minor Allele Frequency of the variant for European Americans in the ESP project</p>
<p><strong>ExAC AF</strong>
Raw allele frequency (population independent) of the variant based on ExAC exomes (AF)</p>
<p><strong>ExAC Het</strong>
The number of heterozygote genotypes observed in ExAC. Pulled from the ExAC AC_Het INFO field.</p>
<p><strong>ExAC HOM</strong>
The number of homozygous alt. genotypes observed in ExAC. Pulled from the ExAC AC_Het INFO field.</p>
<p><strong>ExAC EAS AF (%)</strong>
Adjusted allele frequency (population independent) of the variant based on ExAC (Adj_AC/Adj_AN).</p>
</div>
<div class="section" id="proteomic-and-proteic-data">
<h2>Proteomic and Proteic data<a class="headerlink" href="#proteomic-and-proteic-data" title="Permalink to this headline">¶</a></h2>
<p><strong>PDB</strong></p>
<p><strong>Domain</strong></p>
<p><strong>Topology</strong></p>
<p><strong>Open Target</strong></p>
<p><strong>Path. &amp; Biotech</strong></p>
<p><strong>BIOMUTA</strong></p>
<p><strong>Exons</strong></p>
</div>
<div class="section" id="genomeup-predictions">
<h2>GenomeUp Predictions<a class="headerlink" href="#genomeup-predictions" title="Permalink to this headline">¶</a></h2>
<p><strong>AI 2D</strong></p>
<p><strong>AI 3D</strong></p>
<p><strong>Homology Modeling</strong></p>
<p><strong>Docking</strong></p>
<p><strong>Molecular Dynamics</strong></p>
<p><strong>RMSD WTxMT</strong></p>
</div>
</div>


          </div>
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
        <div class="sphinxsidebarwrapper">
<p class="logo">
  <a href="index.html">
    <img class="logo" src="_static/logo.png" alt="Logo"/>
    
  </a>
</p>








<h3>Navigation</h3>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="getting_started.html">Getting started</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Variants annotation fields</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#genomic-and-genetic-data">Genomic and genetic data</a></li>
<li class="toctree-l2"><a class="reference internal" href="#variant-calling-q-r">Variant Calling Q&amp;R</a></li>
<li class="toctree-l2"><a class="reference internal" href="#evidence">Evidence</a></li>
<li class="toctree-l2"><a class="reference internal" href="#effect-prediction">Effect&amp;Prediction</a></li>
<li class="toctree-l2"><a class="reference internal" href="#frequency">Frequency</a></li>
<li class="toctree-l2"><a class="reference internal" href="#proteomic-and-proteic-data">Proteomic and Proteic data</a></li>
<li class="toctree-l2"><a class="reference internal" href="#genomeup-predictions">GenomeUp Predictions</a></li>
</ul>
</li>
</ul>

<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="index.html">Documentation overview</a><ul>
      <li>Previous: <a href="getting_started.html" title="previous chapter">Getting started</a></li>
  </ul></li>
</ul>
</div>
<div id="searchbox" style="display: none" role="search">
  <h3>Quick search</h3>
    <form class="search" action="search.html" method="get">
      <div><input type="text" name="q" /></div>
      <div><input type="submit" value="Go" /></div>
      <input type="hidden" name="check_keywords" value="yes" />
      <input type="hidden" name="area" value="default" />
    </form>
</div>
<script type="text/javascript">$('#searchbox').show(0);</script>
        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &copy;2018, GenomeUp srl.
      
      |
      Powered by <a href="http://sphinx-doc.org/">Sphinx 1.6.6</a>
      &amp; <a href="https://github.com/bitprophet/alabaster">Alabaster 0.7.10</a>
      
      |
      <a href="_sources/annotation.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>